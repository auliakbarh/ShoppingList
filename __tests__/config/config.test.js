import Config from '@/config';

test('configuration test', () => {
  expect(Config.env).toMatchSnapshot();
});
