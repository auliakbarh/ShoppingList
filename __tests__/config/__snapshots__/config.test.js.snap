// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configuration test 1`] = `
Object {
  "environment": "environment",
}
`;
